pyproject.toml

```diff
@@ -3,7 +3,9 @@ requires = ["pdm-backend"]
 build-backend = "pdm.backend"
 
 [project]
-name = "typer"
+# pip install typer-slim: is only the Typer library, without extras
+# pip install typer: is typer-slim[standard], with extras and typer-cli
+name = "typer-slim"
```

Comment:
Perhaps you want to update the documentation to let users know that `typer` actually includes `typer-slim` and other packages?
This PR offuscates that and this change breaks users of [FawltyDeps](https://github.com/tweag/FawltyDeps). We had declared `typer` as a dependency (-> failed), but it's now the project `typer-slim` the one that has a `typer` package.

Reply:
Hi @FCamborda! Could you open an discussion thread with this question? Because I worry that it might otherwise get lost on this merged PR 🙏

Reply:
@svlandeg Sure! Let me know if this needs improvement: https://github.com/tiangolo/typer/discussions/786